fedbiomed/common/constants.py

```diff
@@ -251,7 +251,7 @@ class SecaggElementTypes(_BaseEnum):
         DIFFIE_HELLMAN: one pair of DH key for each node party, public key shared with other node parties
     """
     SERVER_KEY: int = 0
-    DIFFIE_HELLMAN: int = 2
+    DIFFIE_HELLMAN: int = 1
```

Comment:
Don't we have to increase major version of `__messaging_protocol_version__` then ?
The `SecaggRequest` messages will then incompatible values between versions.

(Anyway the protocol major version is increased to 5 in #1211